pkgs/tools/security/nmap/default.nix

```
@@ -27,7 +27,14 @@ in stdenv.mkDerivation rec {
     sha256 = "063fg8adx23l4irrh5kn57hsmi1xvjkar4vm4k6g94ppan4hcyw4";
   };
 
-  patches = ./zenmap.patch;
+  patches = [ ./zenmap.patch ]
+    ++ optional stdenv.cc.isClang (
+      # https://github.com/nmap/nmap/pull/1363
+      fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/nmap/nmap/pull/1363.diff";
```

please use the specific commit, because the PR contents are prone to change
```suggestion
        url = "https://github.com/nmap/nmap/commit/5bbe66f1bd8cbd3718f5805139e2e8139e6849bb.patch";
```

---

Good catch, updated.